oauth2_http/javatests/com/google/auth/oauth2/ExternalAccountCredentialsTest.java

```diff
@@ -202,6 +202,27 @@ public void fromJson_identityPoolCredentialsWorkforce() {
     assertNotNull(credential.getCredentialSource());
   }
 
+  @Test
```

Comment:
Missing refreshAccessToken tests for all of these credentials (in their respective files, not this one)